ihtml/themes/breezy/recovery.tpl

```diff
@@ -40,39 +37,35 @@
     </p>
 
     <br/>
-    <table>
-      {if $show_directory_chooser}
-      <tr>
-        <td>{t}Directory{/t}</td>
-        <td>
-          <select name="server"  title="{t}Directory{/t}">
-            {html_options options=$server_options selected=$server_id}
-          </select>
-        </td>
-      </tr>
-      {/if}
-      <tr>
-       <td><label for="login">{t}Login{/t}</label></td>
-       <td>
-           <i>{$login}</i>
-       </td>
-      </tr>
-      <tr>
-       <td><label for="new_password">{t}New password{/t}</label></td>
-       <td><input type="password" name="new_password" id="new_password" value="" title="{t}New password{/t}" onkeyup="testPasswordCss(document.getElementById('new_password').value);" /></td>
-      </tr>
-      <tr>
-       <td><label for="new_password_repeated">{t}New password repeated{/t}</label></td>
-       <td><input type="password" name="new_password_repeated" id="new_password_repeated" maxlength="40" value="" title="{t}New password repeated{/t}" /></td>
-      </tr>
-      <tr>
-       <td>{t}Password strength{/t}</td>
-       <td>
+    <div style="text-align: center;">
+      <label for="login">
+        <img class="center" src="{$personal_img|escape}" alt="{t}Username{/t}" title="{t}Username{/t}"/>&nbsp;
+      </label>
+      <input type="text" name="login" value="{$login}" title="{t}Username{/t}" readonly />
+      <br />
+      <label for="new_password">
+        <img class="center" src="{$password_img|escape}" alt="{t}New password{/t}" title="{t}New password{/t}"/>&nbsp;
+      </label>
+      <input type="password" name="new_password" id="new_password" value="" title="{t}New password{/t}" onkeyup="testPasswordCss(document.getElementById('new_password').value);" />
+      <br />
+      <label for="new_password_repeated">
+        <img class="center" src="{$password_img|escape}" alt="{t}New password repeated{/t}" title="{t}New password repeated{/t}"/>&nbsp;
+      </label>
+      <input type="password" name="new_password_repeated" id="new_password_repeated" maxlength="40" value="" title="{t}New password repeated{/t}" />
+      <br>
+      <div style="width: 190px; display: inline-block; margin-left: 54px;" title="{t}Password strength{/t}">
         <span id="meterEmpty" style="padding:0;margin:0;width:100%;background-color:#DC143C;display:block;height:5px;">
         <span id="meterFull" style="padding:0;margin:0;z-index:100;width:0;background-color:#006400;display:block;height:5px;"></span></span>
-       </td>
-      </tr>
-    </table>
+      </div>
+    </div>
+    {if $show_directory_chooser}
+    <div>
+      <img class="center" src="{$directory_img|escape}" alt="{t}Directory{/t}" title="{t}Directory{/t}" />&nbsp;
+      <select name="server" title="{t}Directory{/t}" onchange="javascript:document.loginform.submit();">
+        {html_options options=$server_options selected=$server_id}
+      </select>
+    </div>
+    {/if}
```

Comment:
I will have to test these changes live, but if I understand them correctly they get rid of table to use divs and they make recovery process close to the index login form, which is not a bad idea as recovery is a public page it might as well match style from login which is the other public page.